src/utils.js

```diff
@@ -0,0 +1,45 @@
+const localData = window.localStorage;
+
+function Task(description, completed, index) {
+  this.description = description;
+  this.completed = completed;
+  this.index = index;
+}
+
+const updateLocalStorage = (data) => {
+  data.forEach((itemTask, index) => {
+    itemTask.index = index + 1;
+  });
```

Comment:
- [ ] As per the project requirements `all changes in the to-do list should be updated in the local storage`.  But I can see when I `check mark` a description it should be changed to `true` in the `localstorage` but it is still `false`.

- ![image](https://github.com/kambereBr/ToDoList/assets/60083437/f6220bbe-0e02-455a-aa92-021f9e66ba0f)

Reply:
Thanks @Tufahel for your review